lib/src/text-input/TextInput.test.js

```diff
@@ -643,6 +645,54 @@ describe("TextInput component synchronous autosuggest tests", () => {
     expect(input.value).toBe("");
     expect(queryByRole("listbox")).toBeFalsy();
   });
+
+  test("Autosuggest escapes special characters", () => {
+    const onChange = jest.fn();
+    const { getAllByText, getByText, getByRole } = render(
+      <DxcTextInput label="Autocomplete Countries" suggestions={special_characters} onChange={onChange} />
```

Comment:
Change "special_characters" to camel case.

Reply:
Done